ingenialink/ethernet/servo.py

```diff
@@ -45,6 +98,16 @@ def __init__(self, socket,
         self.socket = socket
         self.ip_address, self.port = self.socket.getpeername()
         super(EthernetServo, self).__init__(self.ip_address)
+        self.__monitoring_num_mapped_registers = 0
+        self.__monitoring_channels_size = {}
+        self.__monitoring_channels_dtype = {}
+        self.__monitoring_data = []
+        self.__processed_monitoring_data = []
+        self.__disturbance_num_mapped_registers = 0
+        self.__disturbance_channels_size = {}
+        self.__disturbance_channels_dtype = {}
+        self.__disturbance_data_size = 0
+        self.__disturbance_data = bytearray()
```

Comment:
Why do you save the monitoring and disturbance status or data?

Reply:
To have the same implementation as IPBServo.